src/_includes/youtube-logo.html

```diff
@@ -0,0 +1,43 @@
+<svg
+  xmlns:dc="http://purl.org/dc/elements/1.1/"
+  xmlns:cc="http://creativecommons.org/ns#"
+  xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  xmlns:svg="http://www.w3.org/2000/svg"
+  xmlns="http://www.w3.org/2000/svg"
+  viewBox="0 0 253.83203 253.83008"
+  width="3em"
+  xml:space="preserve"
+  id="svg2"
```

Comment:
clean up the ids on this. If you want, use a tool called `svgo`, which will help minimize SVGs. Be aware that it actually **changes semantics** on some SVGs, so it may remove the animation class for anime.js for example.

Reply:
only use `svgo` for the ones that you're actually changing.

Reply:
Hm, ok!! I tried to remove the ids but I missed this one. I can test svgo in another time